src/commonComponent/CounterContainer/CounterContainer.jsx

```diff
@@ -1,53 +1,51 @@
 import React, { Component } from 'react';
+import Counter from '../Counter/Counter';
 import './styles.css';
-import Counter from '../Counter/Counter'
-
-class Container extends Component {
 
+class CounterContainer extends Component {
   constructor(props) {
-
     super(props);
 
     this.state = {
 
       countValue: 0,
       parityType: 'The number is even'
-    }
 
+    }
+    
   }
 
-  plusOne = () => {
+  handlePlusOne = () => {
     this.setState({ countValue: this.state.countValue + 1 })
   }
 
-  minusOne = () => {
+  handleMinusOne = () => {
     this.setState({ countValue: this.state.countValue - 1 })
   }
 
-  resetCounter = () => {
+  handleResetCounter = () => {
     this.setState({ countValue: 0 })
   }
 
-  checkParity = () => {
-    this.setState({ parityType: this.state.countValue % 2 === 0 ? 'The number is even' : 'The number is odd' })
-  }
-
   render() {
+
+    const checkParityResult = this.state.countValue % 2 === 0 ? 'The number is even' : 'The number is odd';
+    this.state.parityType = checkParityResult;
```

Comment:
At the previous class I told that "Never  change state directly". For changing state you have setState() method.